### DISCUSSION TITLE
web gui address?

### DISCUSSION THREAD

**Discussion by waipeinan** [benzino77/tasmocompiler#241] [Q&A] [answered]
i am stuck in silly school kid loop here trying to find the url for tasmocompiler
not http://localhost:3000 it seems.

thanks

**Comment by benzino77** [comment, marked as answer]
How do you start TasmoCompiler? Docker? Gitpod? Locally from sources?

**Reply by waipeinan** [reply]
Hiya, I'm trying through gitpod.

**Reply by benzino77** [reply]
Have a look at the picture and description here: [link](https://github.com/benzino77/tasmocompiler#how-to-start-using-tasmocompiler)

**Reply by waipeinan** [reply]
Thank you Benzino77, I am a little dyslexic and despite scanning the' how to' several times, I still managed to miss the tip to disable pop up blockers.  if I had known that the gitpod page was expected to automatically open TasmoComplier I would have worked it out.

Now it's working I cannot thank you enough for this almost foolproof method. 

ta very.  WPN
